spec/models/iiif_content_search_response_spec.rb

```diff
@@ -69,6 +73,12 @@
                                                              "on": 'https://purl.stanford.edu/x/iiif/canvas/y#xywh=0,0,0,0')
     end
 
+    it 'does not include matches that were not returned with <em> tags (intermediate matches)' do
+      response.as_json[:resources].each do |resource|
```

Comment:
Instead of checking what /isn't/ there, it might be more effective to check for only what we think should be there.